packages/tsconfig/adapters.json

```diff
@@ -3,10 +3,6 @@
   "compilerOptions": {
     "target": "ES2019",
     "module": "commonjs",
-    "declaration": true,
```

Comment:
I think we can remove these configs because the tsconfig will be merged with `base.json`. Let's see if it breaks the build 🤔